pyfar/dsp/dsp.py

```diff
@@ -1988,7 +1990,7 @@ def average(signal, mode='linear', caxis=None, weights=None, keepdims=False,
                                         'power',)):
 
         raise ValueError((
-            f"mode '{mode}' is not defined for complex signals."))
+            f"mode '{mode}' is not implemented for complex signals."))
```

Comment:
Maybe you can dicscuss it on Friday, I'm not sure, but I think its not nessesary, since all mentioned options are also working for complex signals...

Reply:
Sorry for that. I already removed that line the previous commit, but something went wrong. Same with the leftovers you found the last time...
You are right, we don't need this errors anymore.